site/.build.js

```diff
@@ -2,22 +2,36 @@ const fs = require('fs');
 const path = require('path');
 const module_path = path.join(__dirname, 'ts');
 
-function getAllFiles(dir) {
+function getTwigTSFiles(dir) {
     return fs.readdirSync(dir, { withFileTypes: true }).reduce((acc, entry) => {
-        if (!entry.isDirectory() && entry.name.endsWith('.ts') || entry.name.endsWith('.js')) {
+        if (entry.isDirectory()) {
```

Comment:
This is related to Matt's comment about the file structure but I think this basically assumes every file under `twig` is an entry point since they will all be given to esbuild to bundle - even if a file is only used as a utility. Not sure if this is what we want since it might cause additional files in the `mjs` dir, but if every file in that directory is an entry point then maybe we can just bring it all up to the top level like @MasterOdin suggested ? 

Not sure how much mirroring the file structure helps when figuring out which file is associated with which twig template since at the end of the day its the developer who decides which `mjs` files are delivered to the user